build/avalanchego-apis/platform-chain-p-chain-api.md

```diff
@@ -1540,6 +1540,57 @@ This gives response:
 }
 ```
 
+### platform.getValidatorsAt
+
+Get the validators and their weights of a subnet or the Primary Network at a given P-Chain height.
+
+#### **Signature**
+
+```cpp
+platform.getValidatorsAt(
+    {
+        height: int,
+        subnetID: string, // optional
+    }
+)
+```
+
+* `height` is the P-Chain height to get the validator set at.
+* `subnetID` is the subnet ID to get the validator set of. If not given, gets validator set of the Primary Network.
+
+#### **Example Call**
+
+```sh
+curl -X POST --data '{
+    "jsonrpc": "2.0",
+    "method": "platform.getCurrentValidators",
```

Comment:
This currently says `getCurrentValidators`. It should be `getValidatorsAt`

Reply:
:facepalm: fixed, thanks!